src/lib/sdk.ts

```diff
@@ -99,36 +129,42 @@ class FlutterSDK {
     }
   }
 
-  private async initDarkSdkHomeFromFlutter(flutterLookup: string) {
+  private async initDarkSdkHomeFromSearchPaths() {
     try {
-      let flutterPath: string;
+      const sdks = await this.findSdks();
+      if (sdks.length > 0) {
+        this._sdkHome = sdks[0].location;
+        await this.initFlutterCommandsFromSdkHome();
+      }
+    } catch(error) {
+      log(`Error configuring sdk from searchPaths: ${error.message}}`);
+    }
+  }
 
-      if (flutterLookup.length == 0) {
-        flutterPath = (await which('flutter')).trim();
+  private async initDartSdkHomeFromLocalFvm() {
+    try {
+      if (await exists('./.fvm/flutter_sdk')) {
```

Comment:
It will has problem if current path is not the project root.

Reply:
Hmm that’s true. Do we already have a way to find the project root ? I would guess it’s available through the workspace like the config?

Reply:
We can get workspace folder from `workspace.workspaceFolder`